packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -2203,4 +2203,20 @@ describe('RpcContractProvider test', () => {
       done();
     });
   });
-});
+  describe('map typecheck error', () => {
+
+    it('should have defined storage with TicketTokens without errors (#1762)', async (done) => {
```

Comment:
I think `TicketTokens` in the title is wrong. Is it not to test Bigmap inside map?

Reply:
right! forgot to change after copy paste